applications/forms.py

```diff
@@ -12,6 +12,7 @@ def __init__(self, *args, **kwargs):
         # which uses choices.
         radio_fields = [
             "all_applicants_completed_getting_started",
+            "read_analytic_methods_policy",
```

Comment:
This! 😱

Reply:
IKR, took me a second to work out why this was needed

Reply:
It made me very sad.